acquia_lift.admin.inc

```diff
@@ -1081,8 +1088,9 @@ function acquia_lift_report_conversion() {
   }
 
   $reports = array();
-  if ($plugin = personalize_agent_load_agent($agent_name)) {
-    $reports = $plugin->buildConversionReport($options);
+  $plugin = personalize_agent_load_agent($agent_name);
```

Comment:
OK, how about this https://github.com/acquia/acquia_lift/commit/93a1450f91dd8857270185aedca670523e519a33